### DISCUSSION TITLE
Error on Sync

### DISCUSSION THREAD

**Discussion by jameslmartin95** [derekantrican/GAS-ICS-Sync#411] [Q&A] [answered]
I just installed this script and attempted to run it. Got this error message: 
<img width="488" alt="image" src="https://github.com/derekantrican/GAS-ICS-Sync/assets/53343966/8545fb90-5b38-45f6-891d-50744f3b482c">
I am using ical to sync my zoho calendar to google calendar, I need it synced every minute. This is the URL Zoho gave to put in Google Calendar.  https://calendar.zoho.com/ical/zz0801123059b4c03090f3c5f56a33c6e6ca856c8eb7e82f82326b76852448bfc99f2027d58114e7d9a2d12e860c8b221be4d08ff3
I am not the most techy person on here, but would appreciate any help.

**Comment by Lonestarjeepin** [comment, marked as answer]
It looks like you've left the "example" calendars in the script.

  ["icsUrl1", "targetCalendar1"],
  ["icsUrl2", "targetCalendar2"],
  ["icsUrl3", "targetCalendar1"]

You should instead replace these with your URL and the target calendar name and delete any that aren't used as well as the comma at the end if you are only using one calendar rather than multiple.

**Comment by jameslmartin95** [comment]
Thanks for that help, I got just a step further. I am attempting to sync Zoho Calendar to Google Calendar. Once I get that working, would like to do a two way sync if possible.

<img width="478" alt="Error Log" src="https://github.com/derekantrican/GAS-ICS-Sync/assets/53343966/01246ff8-3ae8-4644-932f-a64a22e743d0">
<img width="1553" alt="Settings" src="https://github.com/derekantrican/GAS-ICS-Sync/assets/53343966/2862c505-a4a6-4fc6-b391-90c79d486169">


See attached screenshots.




James



---- On Wed, 07 Feb 2024 10:17:51 -0500 Lonestarjeepin ***@***.***> wrote ---





It looks like you've left the "example" calendars in the script.

["icsUrl1", "targetCalendar1"],
 ["icsUrl2", "targetCalendar2"],
 ["icsUrl3", "targetCalendar1"]
You should instead replace these with your URL and the target calendar name and delete any that aren't used as well as the comma at the end if you are only using one calendar rather than multiple.

**Reply by jonas0b1011001** [reply]
It has to be `var sourceCalendars = [["https://calendars.zoho.com/...", "Name of calendar in GCal"]];`

Two way sync is not supported by this project.
